examples/toy/Ch4/mlir/ToyCombine.td

```diff
@@ -0,0 +1,26 @@
+
+#ifdef TOY_COMBINE
+#else
+#define TOY_COMBINE
+
+#ifdef OP_BASE
+#else
+include "mlir/IR/OpBase.td"
+include "include/toy/Ops.td"
+#endif // OP_BASE
+
```

Comment:
Can you provide a bit more documentation on what is happening in this file? This is intended for beginners so we should describe exactly what we are doing at each step.

Reply:
Added more comments that should clarify things. I do have an additional description in the .md file.

Reply:
The code is a tutorial in itself, so we need to make sure that everything is documented as such.

Reply:
Can you update the header blocks to be the same used elsewhere?
https://github.com/tensorflow/mlir/blob/5d81613258e3f76b02246ce772a17359dc069bdf/include/mlir/IR/Block.h#L32